### ISSUE TITLE
Long variable definition with typing format different than black  

### ISSUE THREAD

**Issue by iurisilvio** [astral-sh/ruff#8467]
This code pass black check, but `ruff format` split it in a weird way:

```python
class MyReallyLongClassNameThatDoesNothing:
    pass


long_var_name_to_hit_88_chars: MyReallyLongClassNameThatDoesNothing = (
    MyReallyLongClassNameThatDoesNothing()
)
```

```diff
-long_var_name_to_hit_88_chars: MyReallyLongClassNameThatDoesNothing = (
-    MyReallyLongClassNameThatDoesNothing()
-)
+long_var_name_to_hit_88_chars: (
+    MyReallyLongClassNameThatDoesNothing
+) = MyReallyLongClassNameThatDoesNothing()
1 file would be reformatted
```

This typing is of course not required for my example, but I have many of these happening in a private large codebase.

I'm running ruff from main.

**Comment by charliermarsh** [comment]
Can you confirm that you're on v0.1.3? I believe we fixed this in v0.1.3, and I see the same on the [playground](https://play.ruff.rs/7b18c277-1a1d-4c2d-b6aa-5f1fe224b184):

<img width="1792" alt="Screen Shot 2023-11-03 at 9 48 54 AM" src="https://github.com/astral-sh/ruff/assets/1309177/0e2ba7ad-e866-4291-a99f-89db495df408">

Note that the fix was _not_ retroactive. So if you ran Ruff v0.1.2 on your codebase, and it formatted your code in this way, Ruff v0.1.3 won't change it _back_. But if you remove the parentheses around the annotation, v0.1.3 won't re-add them (i.e., it follows Black's styling).

**Comment by iurisilvio** [comment]
Oops, I was running my own outdated main branch. It is good on upstream main. 🙏🏻 Thanks!
